server/package.json

```diff
@@ -31,6 +31,7 @@
     "version": "standard-version",
     "reset": "git clean -dfx && git reset --hard && npm i",
     "clean": "trash build test",
+    "jest": "jest",
```

Comment:
Why this is defined separately ? Can we not reuse `test` above ?

Reply:
Merged into test , Need to run "npm run test" only